content/en/service_catalog/service_definitions.md

```diff
@@ -38,8 +38,139 @@ For more information about the latest updates, see the schemas on GitHub.
 ### Metadata Schema v3.0 (beta) 
 The Entity Definition Schema is a structure that contains basic information about an entity. See the [full schema on GitHub][1].
 
+#### Example 
+
+##### YAML for `kind:application`
```

Comment:
Are you intending for there to be additional examples added down the road? Otherwise, I would suggest removing the "YAML for `kind:application`" header and combining it with the "Example" header. So something like "Example YAML for `kind:application`"